### DISCUSSION TITLE
Save link map node positions

### DISCUSSION THREAD

**Discussion by payrim** [TriliumNext/Trilium#1831] [Q&A] [answered]
Saving link map node positions so that you could arrange them in a way to create small roadmap and graphs

**Comment by zadam** [comment, marked as answer]
Hi, for that there's https://github.com/zadam/trilium/wiki/Relation-map

Link map on the other hand is auto generated and the positions are not saved.

**Reply by payrim** [reply]
can you define a relation without specifying relation name?

**Reply by zadam** [reply]
No, relation name is mandatory. If you're not interested in the name, just use "x" or some other dummy text ...
